client/src/components/License/LicenseSelector.vue

```diff
@@ -15,17 +21,21 @@
             <a href="#" @click.prevent="editLicense = false">cancel edit.</a>
         </div>
     </div>
-    <div v-else-if="license">
+    <div v-else-if="license" data-description="license selector" :data-license="license">
         <License :license-id="license">
             <template v-slot:buttons>
                 <span v-b-tooltip.hover title="Edit License"
-                    ><font-awesome-icon icon="edit" @click="editLicense = true"
+                    ><font-awesome-icon icon="edit" data-description="edit license link" @click="editLicense = true"
                 /></span>
             </template>
         </License>
     </div>
-    <div v-else>
-        <i><a href="#" @click.prevent="editLicense = true">Specify a license for this workflow.</a></i>
+    <div v-else data-description="license selector" data-license="null">
+        <i
+            ><a href="#" data-description="edit license link" @click.prevent="editLicense = true"
```

Comment:
Oh, prettier.

Reply:
fwiw if you adjust it to the following, prettier will leave it be

```
        <i>
            <a href="#" data-description="edit license link" @click.prevent="editLicense = true">
                Specify a license for this workflow.
            </a>
        </i>
```

Reply:
Yep.  The only thing to be aware of there is that those kinds of workarounds can add significant whitespace depending on the surrounding elements.  I think in this case it'd be fine, but it's not a big deal either way -- just a drive-by comment while reviewing :)